Update set-up-multi-factor-authentication.md (#2472)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md

1. On the **Active users** page, choose **Multi-factor authentication**.
1. On the multi-factor authentication page, select each user and set their Multi-Factor auth status to **Disabled**.

> ### Turn On Per person MFA

Reviewer: ```suggestion
### Turn on per-user MFA
```

> If you have security defaults turned Off and donot want to turn on MFA for the entire organization

Reviewer: ```suggestion
If you have security defaults turned off and do not want to turn on MFA for the entire organization:
```

> 1. In the Microsoft 365 admin center, in the left nav choose **Users** > **Active users**. 

Reviewer: ```suggestion
1. In the Microsoft 365 admin center, in the left nav, choose **Users** > **Active users**. 
```
